recipes/b-multi/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conan import ConanFile
+from conan.tools.files import copy
+
+
+class MultiConan(ConanFile):
+    name = "b-multi"
+    homepage = "https://gitlab.com/correaa/boost-multi"
+    description = "Multidimensional array access to contiguous or regularly contiguous memory. (Not an official Boost library)"
+    topics = (
+        "array",
+        "multidimensional",
+        "library",
+    )
+    license = "Boost"
+    url = "https://gitlab.com/correaa/boost-multi"
+    # No settings/options are necessary, this is header only
+    exports_sources = "include/*"
+    no_copy_source = True
+
```

Comment:
```suggestion

	@property
    def _min_cppstd(self):
        return 17

    @property
    def _compilers_minimum_version(self):
        return {"gcc": "8",
                "clang": "7",
                "apple-clang": "12",
                "Visual Studio": "16",
                "msvc": "192"}

    def layout(self):
        basic_layout(self, src_folder="src")

    def validate(self):
        if self.settings.compiler.cppstd:
            check_min_cppstd(self, self._min_cppstd)
        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
            raise ConanInvalidConfiguration(
                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
            )

    def source(self):
        get(self, **self.conan_data["sources"][self.version], strip_root=True)

```

- Adjust the source folder to follow Conan folder layout
- Use gitlab url to download the sources
- The project requires C++17 (https://gitlab.com/correaa/boost-multi/-/blob/master/CMakeLists.txt?ref_type=heads#L58)

Reply:
I appreciate the fixes, I couldn't have come up with them myself at this stage.